models/labels/addresses/infrastructure/identifier/eth_stakers/labels_eth_stakers.sql

```diff
@@ -11,31 +11,32 @@ WITH identified_stakers AS (
     , 'infrastructure' AS category
     , 'hildobby' AS contributor
     , 'query' AS source
-    , timestamp('2023-01-18') AS created_at
+    , DATE '2023-01-18'  AS created_at
```

Comment:
This may cause timestamp(0) or timestamp(3) error in dune sql engine.

Reply:
Function `timestamp` requires hours and minutes in dune sql engine, which is completely different in spark sql engine. I discussed with Alex earlier to change all `timestamp` without hours and minutes into `date` to avoid errors.

Docs: 
https://trino.io/docs/current/functions/datetime.html

Sample error message:
```
message="Unsupported type: timestamp(0)"
```

Reply:
Could you link to a specific query for this? `timestamp '2023-01-01'` should work.

Reply:
We have a discussion here:
https://github.com/duneanalytics/spellbook/pull/3737

Reply:
same comment here about timestamp function above